src/class-wp-redis-object-cache.php

```diff
@@ -139,7 +139,7 @@ public function __construct($fail_gracefully = true)
             'client' => 'pecl'
         ];
 
-        array_map(function ($setting) {
+        array_map(function ($setting) use(&$parameters){
```

Comment:
Nice catch indeed. Little styling suggestion:

```suggestion
        array_map(function ($setting) use($parameters) {
```